src/app/features/courses/courses.component.ts

```diff
@@ -1,15 +1,60 @@
 import {Component, OnInit} from '@angular/core';
+import { ICourse, Course } from './models/courses.model';
 
 @Component({
-  selector: 'app-courses',
-  templateUrl: './courses.component.html',
-  styleUrls: ['./courses.component.scss']
+    selector: 'app-courses',
+    templateUrl: './courses.component.html',
+    styleUrls: ['./courses.component.scss']
 })
 
 export class CoursesComponent implements OnInit {
```

Comment:
accoring to task 'Assign fake values onngOnInithook.' move this array to some method (for example getCourses()) and call this one in 'ngOnInit' because of in real application we should make a request for data and create method for this one.